src/Config/RuntimeConfigLoader.cs

```diff
@@ -97,7 +93,6 @@ public static bool TryParseConfig(string json,
         ILogger? logger = null,
         string? connectionString = null,
         bool replaceEnvVar = false,
-        string dataSourceName = "",
```

Comment:
`string dataSourceName = "",`
If this has been removed, we should modify the doc string